### ISSUE TITLE
[feature] skip a visual test

### ISSUE THREAD

**Issue by DimitarChristoff** [unindented/react-styleguidist-visual#31]
So, I have a few components that use spinning animations and every time the test runs, it catches the spinner at a different phase, causing false positives. 

I would like to be able to skip an example or a particular element from being considered in the screenshots.

**Comment by unindented** [comment]
The other option (the one we use in our current project) is to disable animations by injecting a CSS file with the following contents:

```css
* {
  animation: none !important;
  transition: none !important;
}
```

**Comment by kot-lex** [comment]
I guess this could be possibly implemented by adding custom action type named "skip"

**Comment by jkarttunen** [comment]
I think the filter argument is for that:
https://github.com/unindented/react-styleguidist-visual/blob/master/src/commands/test.js
